jobs/v3/api_client/requirements-test.txt

```diff
@@ -1,3 +1,4 @@
 pytest==5.3.2
 gcp-devrel-py-tools==0.0.15
+google-cloud-core
```

Comment:
How is google-cloud-core being used? I'm not seeing it anywhere. I could easily be missing something though

Reply:
It is used by gcp-devrel-py-tools, which has a broken dependency.
See: https://github.com/GoogleCloudPlatform/python-repo-tools/issues/24

I'll fix them with upcoming PR.